parser.y

```diff
@@ -8144,10 +8179,27 @@ NationalOpt:
 	{}
```

Comment:
is `NationalOpt` necessary now?

Reply:
it comes from the master. I have no idea what it is.

Reply:
remove it

Reply:
it was only used in `StringType`, but it's unused now

Reply:
oh, i am sorry, i mistake this pr into #465, this bug should be fixed in #465, but it has been merged...
could you help me fix it by the way?

Reply:
My bad, it comes from the auto-merge. Should have check manually. Fixed